app/views/planning_applications/validation/cil_liability/_form.html.erb

```diff
@@ -16,5 +16,8 @@
     <% end %>
   <% end %>
 
-  <%= render(partial: "shared/submit_buttons", locals: {form:}) %>
```

Comment:
It's a shame that we can't keep using the partial when we don't want the 'save and come back later' button.

I wonder if we could turn this into a view component with options for which buttons to show. (Not for this PR though.)